raiden/ui/web/src/app/components/channel-table/channel-table.component.html

```diff
@@ -7,7 +7,7 @@
     </div>
     <p-dataTable [value]="channels" [paginator]="true" [rows]="10" [resizableColumns]="true" [globalFilter]="gb"
                  dataKey="channel_identifier" [rowTrackBy]="rowTrackBy">
-      <p-column field="channel_identifier" header="Channel" [sortable]="true">
+      <p-column field="channel_identifier" header="Channel" [sortable]="true" [style]="{width: '8em'}">
```

Comment:
Not important, but just a headsup that, for styling single properties, you can do something like `[style.width.em]="8"`, or of course, as a static property, declare it statically too, to leave it out of the changes detectors and rendering engine.

Reply:
I did try it after reading the comment and `[style.width.em]="8"` doesn't seem to have the same effect as `[style]="{width: '8em'}` for some reason.
I will leave it as it is for the moment and re-evaluate in the future.